lib/modules/apostrophe-assets/public/js/popups/tags.js

```diff
@@ -53,10 +58,8 @@ $(function () {
     var $clone = $tag.clone();
     $clone.removeAttr('data-tag-picker-selected');
     $clone.on('click', function (e) {
-      var $this = $(this);
       e.preventDefault();
-      $context.find('[data-tag-picker-selected]button[value="' + $this.val() + '"]').attr('data-tag-picker-selected', 'false');
-      $this.remove();
+      removeTag($clone, $context);
```

Comment:
Without this the implementation was incomplete, it was visually removed on click but not removed from the list on submit. Also hey less code and seems to work?